script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_5B.txt

```diff
@@ -98,55 +98,55 @@ All I want right now is to be left alone in my room, take off my glasses, close
 @gThe woman swimming in the crowd is beautiful, both pure and desirable.
 <Page20>
 "―――――――"
-@gMy throat is aching.
-@gI haven't been able to breathe properly for a while now.
+@gMy THROAT is ACHING.//@g　ノドがアツイ。katakana (candidate for font/italics change)
+@gI haven't been ABLE to BREATHE for a WHILE now.//@g　サッキからイキがぜんぜんデキナイ。 (candidate for font/italics change)
 @gWhat's wrong with that? It's only natural.
-@gAfter witnessing such a ■, it would be rude for a creature to not get excited.
+@gAfter witnessing such a ■, it would be rude for a creature not to get excited.
 <Page21>
 @gI take something out of my bag.
 @gI put it in my pocket and start to walk.
 @gI feel the touch of iron on my fingertips.
 @gI am truly blessed.
 @gThis tool is just perfect.
 <Page22>
-@gHow wonderful. I'm going to come. What a sensual encounter.
+@gHow wonderful. I'm going to climax. What a sensual rendezvous.//@g　すばらしい。いきそうだ。なんて官能的な逢瀬。
 @gI must keep a good distance.
-@gDon't be noticed.
-@gDon't let anyone around us get in the way.
+@gSo as not to be noticed.//@g　気付かれないように、
+@gSo as not to arouse the suspicions of the nuisances that surround us.//@g　周りの邪魔者に気取られないように。
 @gThat woman and I are complete strangers.
-@gSo I have to follow her scent as naturally as possible.
+@gSo, as naturally as possible, I have to follow her scent.//@g　だから出来るだけ自然に、匂いを辿っていかなければ。
 <Page23>
 @gThe woman enters an apartment complex.
 @gAn automatic glass door. The security is perfect.
-@gNo one can even enter the hall unless they're residents.
-@g I go around to the emergency exit, take off my jacket and leave it somewhere hidden. I cut. And go inside.
+@gNo one can even enter the hall unless they're a resident.
+@g I go around to the emergency exit, take off my jacket, and leave it somewhere hidden. I cut. And I go inside.
```

Comment:
Is there supposed to be a space at the start of the line? 
Maybe drop the `And` on the final sentence if we have the `I` there?

Reply:
The And was added intentionally, reading it aloud it sounded weird to me without it.

Reply:
on second thought, I'm gonna drop it again